### PR TITLE
Add TimeBasedMedia fields

### PR DESCRIPTION
**Pull request by mikejritter**
Add fields and layout defined in:
* https://collectionspace.atlassian.net/browse/DRYD-1034
* https://collectionspace.atlassian.net/browse/DRYD-1039
* https://collectionspace.atlassian.net/browse/DRYD-1040
* https://collectionspace.atlassian.net/browse/DRYD-1041

### CODE REVIEW

**Review by ray-lee — CHANGES_REQUESTED** [review]
This is mostly good, but the main thing is missing `fullName` messages, and redundant `name` messages. A couple tips:
- Review the [style guide](https://github.com/collectionspace/cspace-ui.js/blob/master/docs/style/README.md).
- When testing your changes, look at what happens on the Search page. Select the record type you're working on, and click the **+ Field** button to add a field to search. If anything in the field selection dropdown is ambiguous, you likely need to add `fullName` messages to disambiguate. For example, the "Version" fields here:
search_fields
Conventionally, fields in a group should be grouped together in that dropdown, meaning that their `fullName` messages should all start with the same prefix. The convention is that the `fullName` of a field in a group is the `name` of the group + the `name` of the field.

File: src/plugins/recordTypes/collectionobject/fields.js [hunk 1]
```diff
@@ -3994,6 +3994,1209 @@ export default (configContext) => {
...
+          },
+          colorSpaceGroup: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.collectionobjects_common.colorSpaceGroup.name',
+                  defaultMessage: 'Color space',
+                },
+              }),
+              repeating: false,
```
[the remark this answers is not in the document]
Comment (mikejritter) [reply src/plugins/recordTypes/collectionobject/fields.js:5134]: @ray-lee  Looking back at the specs, the way it's formatted in the spreadsheet made me think it was supposed to be a group, but the wireframes didn't show them as being repeatable. Probably something to confirm with Megan about.

I also just saw that a few fields (`audioType`, `audioPreferences`, `audioSpecificationNotes`) are marked as being repeatable in the spreadsheet as well... maybe I should update everything to match the wireframes instead.

[the remark this answers is not in the document]
Comment (mikejritter) [reply src/plugins/recordTypes/collectionobject/fields.js]: Updated to be repeating

File: src/plugins/recordTypes/collectionobject/fields.js [hunk 2]
```diff
@@ -3994,6 +3994,1209 @@ export default (configContext) => {
...
+          },
+          aspectRatioGroup: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.collectionobjects_common.aspectRatioGroup.name',
+                  defaultMessage: 'Aspect ratio',
+                },
+              }),
+              repeating: false,
```
[the remark this answers is not in the document]
Comment (mikejritter) [reply src/plugins/recordTypes/collectionobject/fields.js]: Updated to be repeating

**Comment by ray-lee** [comment]
This looks good. I didn't test the full functionality, but that can be done when it's deployed to dev.

**Pull request merged**